nisystemlink/clients/dataframe/_data_frame_client.py

```diff
@@ -17,7 +17,11 @@ def __init__(self, configuration: Optional[core.HttpConfiguration] = None):
         Args:
             configuration: Defines the web server to connect to and information about
                 how to connect. If not provided, an instance of
-                :class:`core.JupyterHttpConfiguration` is used.
+                :class:`JupyterHttpConfiguration <nisystemlink.clients.core.JupyterHttpConfiguration>`
+                is used.
+
+        Raises:
+            ApiException: If unable to communicate with the Data Frame service.
```

Comment:
I don't know if it matters, but tag uses `if the API call fails.` (with a lowercase "if") for all of these. I kind of like what you have more, but figured I would bring it up.

Reply:
I'll update to make them consistent one way or the other.